public/playersBetController.js

```diff
@@ -0,0 +1,140 @@
+    this.addEventListeners();
+  }
+
+  initPlayers() {
+    this.players = this.board.players;
+    this.updateActivePlayer();
+  }
+
+  selectDomElements() {
+    this.betControls = document.querySelector(".bet-controls");
+    this.bet25 = this.controlsContainerEle.querySelector(".btn--25");
+    this.bet50 = this.controlsContainerEle.querySelector(".btn--50");
+    this.bet100 = this.controlsContainerEle.querySelector(".btn--100");
+    this.clear = this.controlsContainerEle.querySelector(".btn--clear");
+    this.currentPotCont =
+      this.controlsContainerEle.querySelector(".bet-controls__pot");
+    this.curretBetCont =
+      this.controlsContainerEle.querySelector(".bet-controls__bet");
+    this.confirm = this.controlsContainerEle.querySelector(".btn--bet");
+  }
+
+  addEventListeners() {
+    this.bet25.addEventListener("click", this.betting.bind(this));
+    this.bet50.addEventListener("click", this.betting.bind(this));
+    this.bet100.addEventListener("click", this.betting.bind(this));
+    this.clear.addEventListener("click", this.clearBet.bind(this));
+    this.confirm.addEventListener("click", this.confirmBet.bind(this));
+  }
+
+  updateActivePlayer() {
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.add("active");
+  }
+
+  isLastPlayerTurn() {
+    const lastPlayerPosition = this.players.length - 2;
+    return lastPlayerPosition === this.currentPlayerTurn;
+  }
+
+  nextPlayer() {
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.remove("active");
+    if (this.isLastPlayerTurn()) {
+      this.betControls.remove();
+      this.board.sartWithGameDealing();
+    } else {
+      this.currentPlayerTurn++;
+      this.updateActivePlayer();
+      this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+      this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+    }
+  }
+
+  betting(e) {
+    this.clear.disabled = false;
+    this.confirm.disabled = false;
+    const betAddition = e.target.innerText;
+    if (this.activePlayer.pot - betAddition < 0) {
+      return;
+    }
+    this.activePlayer.pot -= betAddition;
+    this.currentBet += 1 * betAddition;
+    this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+    this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+  }
+
+  clearBet() {
+    this.clear.disabled = true;
+    this.confirm.disabled = true;
+    this.activePlayer.pot += this.currentBet;
+    this.currentBet = 0;
+    this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+    this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+  }
+
+  confirmBet() {
+    this.clear.disabled = true;
+    this.confirm.disabled = true;
+    let playerHtmlCont = document.querySelector(`#${this.activePlayer.name}`);
+    playerHtmlCont.insertAdjacentHTML(
+      "afterbegin",
+      `Betting: ${this.currentBet}`
```

Comment:
This logic should better be moved inside the player entity so that we can call something like `this.activePlayer.renderBetValue(this.currentBet)`